scalecube-organization/src/main/java/io/scalecube/organization/operation/ServiceOperation.java

```diff
@@ -96,11 +90,10 @@ protected GetOrganizationResponse getOrganizationResponse(
 
   protected OrganizationInfo.Builder organizationInfo(
       Organization organization, Predicate<ApiKey> filter) {
-    ApiKey[] apiKeys = Arrays.stream(organization.apiKeys()).filter(filter).toArray(ApiKey[]::new);
     return OrganizationInfo.builder()
         .id(organization.id())
         .name(organization.name())
-        .apiKeys(apiKeys)
+        .apiKeys(organization.apiKeys().stream().filter(filter).toArray(ApiKey[]::new))
```

Comment:
:+1: that's a better option to make an array, but why do we use arrays and not `List`s?

Reply:
I don't know why it (arrays in `OrganizationInfo) was designed in this way.
But I didn't change it since I avoided changing the public API in this PR.